tools/azure_iot_operations_stub_services/src/schema_registry/service.rs

```diff
@@ -17,10 +17,10 @@ use crate::{OutputDirectoryManager, schema_registry::service_gen};
 use crate::{
     ServiceStateOutputManager,
     schema_registry::{
-        SERVICE_NAME, Schema,
+        ErrorCode, ErrorTarget, SERVICE_NAME, Schema, ServiceError,
         schema_registry_gen::{
-            common_types::common_options::CommandOptionsBuilder,
-            schema_registry::service::{GetResponsePayload, PutResponsePayload},
+            common_types::options::CommandExecutorOptionsBuilder,
```

Comment:
same comment about importing the gen module with an alias (although it looks like you're partially doing that already?)